pkg/textparse/openmetricsparse.go

```diff
@@ -233,7 +233,7 @@ func (p *OpenMetricsParser) Next() (Entry, error) {
 	p.hasExemplarTs = false
 
 	switch t := p.nextToken(); t {
-	case tEofWord:
+	case tEOFWord:
```

Comment:
Shouldn't this be `EOfWord`?

Reply:
Go precedent is `EOF`, as in `io.EOF`.

Reply:
`EOF` is an acronym that stands for "End Of File".

In `EOfWord` only the `E` is an abbreviation, it stands for "End Of Word".

`EOFWord` would suggest "End Of File Word" or "End Of Foo Word" or something.

Reply:
And now I have to correct myself, it actually does stand for "End Of File Word" or "EOF Word". So please forget my whole line of argument here. All good.

Reply:
Ack, it didn't even occur to me that this might not mean "end-of-file". Apologies for jumping in!

Reply:
But it does. You were right, and I was wrong. Thanks for jumping in! :o)